packages/client/components/PokerEstimatePhase.tsx

```diff
@@ -40,13 +38,49 @@ const EstimateAreaWrapper = styled('div')({
   flexDirection: 'column'
 })
 
-interface Props extends PokerMeetingPhaseProps {
+type Props = {
+  toggleSidebar: () => void
+  avatarGroup: ReactElement
   gotoStageId: ReturnType<typeof useGotoStageId>
-  meeting: PokerEstimatePhase_meeting
+  queryRef: PreloadedQuery<any> // TODO: update
 }
 
 const PokerEstimatePhase = (props: Props) => {
-  const {avatarGroup, meeting, toggleSidebar, gotoStageId} = props
+  const {avatarGroup, toggleSidebar, gotoStageId, queryRef} = props
+
+  const data = usePreloadedQuery(
+    graphql`
+      query PokerEstimatePhaseQuery($meetingId: ID!) {
+        viewer {
+          meeting(meetingId: $meetingId) {
+            ... on PokerMeeting {
+              ...EstimatePhaseArea_meeting
+              id
+              endedAt
+              isCommentUnread
+              isRightDrawerOpen
+              localStage {
+                ...PokerEstimateHeaderCard_stage
+              }
+              # phases {
+              #   ... on EstimatePhase {
+              #     stages {
+              #       ...PokerEstimateHeaderCard_stage
```

Comment:
`PokerEstimateHeaderCard_stage` includes expensive fields such as `descriptionHtml`. Rather than query this for every stage, I just want to query this for the `localStage`, which is what the user currently needs